lib/matplotlib/lines.py

```diff
@@ -592,6 +599,61 @@ def get_markevery(self):
         """return the markevery setting"""
         return self._markevery
 
+    @property
+    def downsample(self):
+        """Set the downsample property to subsample the plotted line segments.
+
+        If True, then only a subset of line segments will be plotted. The
+        subset is carefully chosen so that the rendering of the downsampled
+        plot looks very similar to the rending of the non-downsampled plot. In
+        fact, if there is no anti-aliasing and plotting uses a solid line only,
+        then the renderings will be identical.
+
+        Recommended *only* when plotting with a solid line (e.g. no dashes,
+        no markers).
+
+        ACCEPTS: [True | False]
+
+        Parameters
+        ----------
+        downsample: True | False
```

Comment:
can you add a space after the column. It else doesn't render properly with numpydoc.